docs/FAQs.md

```diff
@@ -3,10 +3,63 @@ sidebar_position: 7
+
+**Private** projects(including the underline studies and datasets) are only visible and accessible to you or the members of the team and also to the user with whom you explicitly share the project.
+
+### What happens when I make my project public?
+If you make your project public you are making it open for access(including the underline studies and datasets) to everyone(even to the non-registered user of [nmrXiv](https://nmrxiv.org/)), but you can define rights by choosing a license for your project. Once your project is made public you cannot edit, delete or change it to private again.
+
+### Can I edit my project once made public?
+You cannot edit the project once it's made public, but you can always create another version of the project and make changes on top of it.
+
+### How can I delete my project, studies or dataset?
+
+
+### How can I share my project?
+You can share your project with multiple users. Not only that you can also choose the type of role you want. Click [here](/docs/submission-guides/sharing) to learn more about sharing.
+
+### Can I create a team to work on a collaborative project?
+Yes, you can create a [Team](/docs/submission-guides/team) and add as many members as you want. You can also restrict the action of members by assigning different roles. Click [here](/docs/submission-guides/team) to learn more about Team.
+
+
+### What are the different roles I can assign to a member of a team?
+Below are the three major roles which you assign to a member. 
+* **Owner** - Can read and/or update including deleting the project, study and dataset.
+* **Collaborator** -  Can read and/or update project, study and dataset information.
+* **Reviewer** - Can only read project, study and dataset information.
+
+### How can I delete or edit my account?
+You can edit the basic information in your account by clicking on the `Account` tab from the top right corner drop-down in your dashboard. To delete your account please reach out to our [Helpdesk](https://www.nfdi4chem.de/index.php/helpdesk/) or write to us at info@nmrxiv.org.
+
+### How can I license my project and how to know which license to choose?
+Licensing your project is important before you make it public. You can choose the license from the license dropdown. Please go through this [discussion](https://github.com/NFDI4Chem/nmrxiv/discussions/123) to learn more about which license to choose.
+
+### Can I use nmrXiv for my teaching and demo purposes?
+Yes, you can use our [staging site](https://dev.nmrxiv.org) for teaching, demoing, and also test most of the features. But please keep in mind this site is just for testing and demonstration purposes only and all your data can be reset anytime.
+To learn more about our environment click [here](docs/developer-guides/deployment/environment).
+
+### How to report a bug?
+Before reporting a new issue
+* Please read the [documentation](https://docs.nmrxiv.org/) carefully.
```

Comment:
maybe more specific link? And most importantly to answer how to report security bugs at the beginning (as a link)